### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Documentation Update

## Description

This updates the PR template with questions about tests and post-deployment tasks, as per our recent PR discussion where most people seemed in favor.

## Related Tickets & Documents

n/a

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -26,12 +26,20 @@
 
 ## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
 
+## Addeds tests?
```
Comment (jacobherrington) [inline .github/PULL_REQUEST_TEMPLATE.md]: `Addeds` should be `Added`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -26,12 +26,20 @@
...
+- [ ] no, because they aren't needed
+- [ ] no, because I need help
+
 ## Added to documentation?
 
 - [ ] docs.dev.to
 - [ ] readme
 - [ ] no documentation needed
 
+## [optional] Post deployment tasks
```
Comment (jacobherrington) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'm not sure this is clear enough, should it be a question? For example, "Are there any post deployment tasks we need to perform?"

Your choice on this!

File: docker-compose.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ services:
     depends_on:
       - db
       - redis
+      - elasticsearch
```
Comment (jacobherrington) [inline docker-compose.yml]: :thinking: Should these be here?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (citizen428) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks for catching this @jacobherrington!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (citizen428) [reply .github/PULL_REQUEST_TEMPLATE.md]: Funny, I had a question first, then thought it's too long and shortened it. I take this to mean that my first gut feeling was correct and made it a question again, thanks!

File: docker-compose.yml (hunk 3 above)
Comment (citizen428) [reply docker-compose.yml]: Nope. I was sort of working on two branches at the same time because Docker on macOS is slooooooow and this snuck in.

**Comment by citizen428** [comment]
Thanks @rhymes and @jacobherrington, updated.

**Pull request merged**